libconnman-qt/commondbustypes.cpp

```diff
@@ -8,38 +8,7 @@
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
-#include "libconnman_p.h"
-
-#define CONNMAN_ERROR "net.connman.Error"
-
-const QString ConnmanError::Failed(CONNMAN_ERROR ".Failed");
```

Comment:
I'm not sure if it is a good idea to remove these now. I just thought that these could be used more in passing information upwards about what was wrong with the services, and to react to these better. In many ways the error reporting on WLAN and VPN is quite low on detail and user has to guess what was wrong.

Reply:
Can be relatively easily added back if there's something done to this direction. Now it's just dead code.